static/sw.js

```diff
@@ -0,0 +1,11 @@
+importScripts('/bower_components/sw-toolbox/sw-toolbox.js');
+
+toolbox.options.debug = true;
+toolbox.router.default = toolbox.fastest;
+
+const filesToCache = [
+    '/',
+    'https://fonts.googleapis.com/css?family=Roboto:regular,bold,italic,thin,light,bolditalic,black,medium&lang=en'
```

Comment:
I'm not sure of the intention here but looks like this will only cache the CSS, not the fonts themselves--I don't think this will actually provide fonts offline. (Although it will "work" on modern Android devices, because Roboto is pre-installed.)

Providing fonts offline is quite difficult (especially via Google Fonts) due to font subsetting and other browser-based optimizations that the service worker doesn't have visibility of--are there some other resources that can be cached instead?